channels/microsoft365/m365-graph/index.rst

```diff
@@ -0,0 +1,129 @@
+  - Accounts in any organizational directory (Multitenant)
+
+- Under "Redirect URI", select "Web" as platform and paste your already copied
+  callback URL from Zammad.
+- Click on **Register**.
+- In the overview screen, copy the "Application (client) ID", switch to Zammad
+  and paste it in the "Client ID" field in the pop up.
+- Only required for single tenant setup: copy the "Directory (tenant) ID" and
+  paste it in the "Tenant UUID/Name" field in Zammad.
+
+Create Secret
+^^^^^^^^^^^^^
+
+- In Entra, go to "Certificates & secrets" and add a secret by clicking the
+  **New client secret** button.
+- Enter a description, set an expiry duration and click **Add**.
+- Copy the string under "Value", this is the secret. Paste it to Zammad
+  in the "Client Secret" field.
+
+Configure API Permissions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Go to "API permissions" and **Add a permission**.
+- Select "Microsoft Graph" and "Delegated permissions".
+- Add the following permissions:
+
+  - ``offline_access``
+  - ``openid``
+  - ``profile``
+- Add the following permissions for **user mailboxes** only:
+
+  - ``Mail.ReadWrite``
+  - ``Mail.Send``
+- Add the following permissions for **shared mailboxes** only:
+
+  - ``Mail.ReadWrite.Shared``
+  - ``Mail.Send.Shared``
+
+Save it by clicking the **Add permissions** button.
+
+Enterprise Applications
+^^^^^^^^^^^^^^^^^^^^^^^
+
+- Go to *Applications > Enterprise applications* and select your just created
+  app.
+- Go to "Users and groups" and **Add user/group**.
+- Click "None selected" to add a user.
+- Search and select your user and click on **Select** and then **Assign** to
+  save it.
+
+Additional Steps for Shared Mailboxes
+-------------------------------------
+
+If you want to fetch email from a shared mailbox, you have to perform additional
+steps:
+
+- Log in to the `Exchange admin center <https://admin.exchange.microsoft.com>`_.
+- Go to *Recipients > Mailboxes*, select the mailbox and click on **Mailboxes
+  delegation**.
+- Under "Send as", click on **Edit** and **Add members**. Search and select the
+  user from which you want to access the mailbox and **Save** it.
+- Do the same for "Read and manage (Full Access)".
+
+
+Configure the Channel in Zammad
+-------------------------------
+
+App Configuration
+^^^^^^^^^^^^^^^^^
+
+If you followed this guide, you already should have pasted your app information.
+If not, here is a short summary:
+
+- In Zammad's channel configuration, click on **Configure App**.
+- Enter your app details:
+
+  - Client ID: *Application (client) ID*
+  - Client Secret: *Value* from client secret
+  - Tenant UUID/Name: *Directory (tenant) ID* (not required for shared mailboxes)
+- Click on **Submit**.
+
+Add Account
+^^^^^^^^^^^
+
+Now you can add your account to Zammad. Do so by clicking the **Add account**
+button in the top right corner. Select the correct mailbox type and click
+**Authenticate**. Enter your credentials and confirm the requested permissions
```

Comment:
Perhaps we can mention here that in case of a shared mailbox, its email address *has* to be provided in a separate input box. Also, it will not be validated at this step (beside some basic email address format validation), since the API cannot be used until the authentication flow is successfully completed.